guides/v3.5.0/tutorial/service.md

```diff
@@ -208,25 +168,30 @@ This property will be passed in to the component by its parent template below.
 
 Finally open the template file for our `rental-listing` component and add the new `location-map` component.
 
-```handlebars {data-filename="app/templates/components/rental-listing.hbs" data-diff="+20"}
+```handlebars {data-filename="app/templates/components/rental-listing.hbs" data-diff="+25"}
 <article class="listing">
-  <a {{action 'toggleImageSize'}} class="image {{if isWide "wide"}}">
-    <img src="{{rental.image}}" alt="">
+  <a
+    class="image {{if isWide "wide"}}"
+    onclick={{action "toggleImageSize"}}
+    role="button"
+  >
+    <img src={{rental.image}} alt="">
     <small>View Larger</small>
   </a>
-  <h3>{{rental.title}}</h3>
-  <div class="detail owner">
-    <span>Owner:</span> {{rental.owner}}
-  </div>
-  <div class="detail type">
-    <span>Type:</span> {{rental-property-type rental.category}}
-      - {{rental.category}}
-  </div>
-  <div class="detail location">
-    <span>Location:</span> {{rental.city}}
-  </div>
-  <div class="detail bedrooms">
-    <span>Number of bedrooms:</span> {{rental.bedrooms}}
+  <div class="details">
+    <h3>{{link-to rental.title "rentals.show" rental class=rental.id}}</h3>
```

Comment:
```suggestion
    <h3>{{rental.title}}</h3>
```

The `rentals.show` route doesn't exist at this point in the tutorial, so the `{{link-to}}` breaks the app at this stage of the walkthrough